examples/classic_controllers/custom_classic_controllers_synch_motor_example.py

```diff
@@ -0,0 +1,114 @@
+from classic_controllers import Controller
+from externally_referenced_state_plot import ExternallyReferencedStatePlot
+import gym_electric_motor as gem
+from gym_electric_motor.visualization import MotorDashboard
+import numpy as np
+
+if __name__ == '__main__':
+
+    """
+        motor type:     'PMSM'      Permanent Magnet Synchronous Motor
+                        'SynRM'     Synchronous Reluctance Motor
+                        
+        control type:   'S'         Speed Control
+                        'T'         Torque Control
+                        'C'         Current Control
+
+        action_type:    'AbcCont'   Continuous Action Space in ABC-Coordinates
+                        'Finite'    Discrete Action Space
+    """
+
+    motor_type = 'PMSM'
+    control_type = 'S'
```

Comment:
I think it would be more readable and the structure of the environments naming is displayed better, if the control_type would be changed to SC/TC/CC instead of the single S/T/C at this point and in the other example.